checkov/common/sast/report_types.py

```diff
@@ -34,7 +34,7 @@ class DataFlow(BaseModel):
 
     @model_serializer
     def serialize_model(self) -> Dict[str, List[MatchLocation]]:
-        return {'data_flow': self.data_flow}
+        return {'df': self.data_flow}
```

Comment:
why do we need this change ? isn't it breaking something ?

Reply:
it was a test for me to see if my changes really work, i removed it